remote-build.exp

```diff
@@ -101,6 +100,12 @@ while { ${argi} < ${argc} } {
 				error "Could not parse ${arg}"
 			}
 		}
+		"--dir=[^ ]*" {
```

Comment:
I still think having a 1:1 correspondence would help understanding what each option is doing.